### DISCUSSION TITLE
Independent Editing Mode, Visual Linking, and Timing Transfer

### DISCUSSION THREAD

**Discussion by Ra1nek** [SubtitleEdit/subtitleedit#8753] [Ideas]
**Proposal for New Subtitle Editing Features**

### 1.  New Subtitle Editing Mode:

Implement a new subtitle editing mode similar to the current translation mode, but without mandatory linkage between dialogues in the original and translated subtitles. In this mode, each subtitle should function independently, allowing the user to edit both subtitles separately. This is particularly useful for adapting, splitting lengthy dialogues, or adding new dialogues in each subtitle. The number of dialogues in the two subtitles may differ.

Currently, I use **WinMerge** for comparing and editing subtitles, which is not ideal for quickly aligning subtitles in my native language with the original, especially when the number of dialogues varies significantly.

### 2. Visual Dialogue Linking:
Add a feature that allows users to visually link one or more lines from one subtitle to corresponding lines in another subtitle. This will make it easier to navigate and track how the dialogues match up between the two subtitles. For example, a user could link line **№1** in one subtitle with line **№3** in the other. When scrolling, these linked lines would automatically stay in sync, and the linked lines in both subtitles would be highlighted to clearly show the correspondence.

**Explanation**: For accuracy, there should be an option to link all dialogues starting from the first one, with scrolling synchronized in both subtitles or turned off if not needed. When a specific line is selected in one subtitle and the corresponding line in the other, linking should be established between them. If new lines are linked, old links should be removed, leaving only the most recent links.

### 3. Dialogue Timing Transfer:
Implement functionality for transferring dialogue timings between subtitles. The user should be able to select one or more lines in one subtitle and transfer the corresponding timings to another subtitle, regardless of the specific dialogue text. For example, the user can transfer the timing of dialogue **№1** in one subtitle to dialogue **№1** or **№3** in another subtitle, or transfer the timings for a range of dialogue lines, such as from **№1** to **№120** in one subtitle to lines **№1** to **№120** or **№10** to **№130** in the other subtitle.

**Additional**: Also, retain the ability to adjust timing based on the audio track in this mode. The active subtitle will be the one where the dialogue is selected in the editing window. It is assumed that the video content is the same for both subtitles.

**Comment by EloyFarina** [comment]
Very interesting!!! We had the same idea and we use the same softwares also haha. Thanks for explaining better than me. My natural lenguaje is spanish and my English vocabulario is límited.
I'm commenting here to be updated when someone writes a new comment.
